code/ui/manager/src/components/sidebar/TreeNode.tsx

```diff
@@ -161,7 +142,9 @@ export const GroupNode: FC<
     <BranchNode isExpandable={isExpandable} tabIndex={-1} {...props}>
       <IconsWrapper>
         {isExpandable ? <CollapseIcon isExpanded={isExpanded} /> : null}
-        <TypeIcon icon="folder" useSymbol color="primary" />
+        <TypeIcon viewBox="0 0 14 14" width="14" height="14" type="group">
+          <use href="#icon--group" />
```

Comment:
can these IDs (`#icon--group`) be imported from where the symbols are defined instead?
Currently there's no typed relation between the symbols definition and their usage, so someone could rename this to `#icon-asdf` without getting any errors, which is unfortunate.
You could also define the whole `<use href="#icon-group" />` with the symbols and import that, up to you.

Importing them also tells the next engineer how these are defined and how they work, something that you understandably struggled with when you saw this the first time, so I think this would be a nice improvement.

Reply:
That is much better indeed. I created a component that I import and that is typed with shared enums. You can't mess around now :)